extensions/interactions/CodeRepl/directives/oppia-interactive-code-repl.component.spec.ts

```diff
@@ -151,8 +151,8 @@ describe('InteractiveCodeReplComponent', () => {
   it('should update output when the code completes execution', () => {
     spyOn(currentInteractionService, 'registerCurrentInteraction').and.stub();
     spyOn(Sk, 'configure').and.callFake((
-        obj: {output: Function, read: Function, timeoutMsg: Function,
-          execLimit: number}) => {
+        obj: { output: Function; read: Function; timeoutMsg: Function;
+          execLimit: number; }) => {
```

Comment:
```suggestion
        obj: { 
          output: Function; 
          read: Function; 
          timeoutMsg: Function;
          execLimit: number; 
        }
    ) => {
```

Reply:
@vojtechjelinek This exact same type occurs three times in this file. What if we create an interface for it:
```
interface SkulptConfigureArgs = {
  output: Function;
  read: Funciton;
  timeoutMsg: Function;
  execLimit: number;
}
```
and use it instead?

Reply:
Yes, maybe `SkulptConfigurationArgs`

Reply:
Also maybe look into _oppia-interactive-code-repl.component.ts_ since the type might also be used there.

Reply:
I couldn't find an interface in `oppia-interactive-code-repl.component.ts` that could be used. I have created a new interface used it instead.